site/content/entry/yuzu-progress-report-feb-2023/index.md

```diff
@@ -0,0 +1,377 @@
+The problem was in our Qt UI, and german77 worked to properly tell Qt to save and sync the configuration to file, avoiding any {{< gh-hovercard "9817" "configuration change loss" >}} after a force close.
+
+## General fixes
+
+[MonsterDruide1](https://github.com/MonsterDruide1) continues to do wonders with the [LDN](https://yuzu-emu.org/entry/ldn-is-here/) code.
+This time, proper error handling.
+One of the possible errors when handling TCP connections is `ECONNRESET`, which happens when the other end closes the connection abruptly.
+Old-school gamers call this “rage quitting”.
+
+Jokes aside, when a sudden disconnection like this happens, the error used to get caught by yuzu instead of the game, causing the game to never get the notification of the client disconnection.
+Properly {{< gh-hovercard "9843" "passing the error" >}} through to the game allows it to handle the issue its own way instead of crashing the yuzu LDN session.
+
+Some improvements were done to yuzu-cmd, the command-line alternative to the default Qt UI.
+
+german77 added support for {{< gh-hovercard "9729" "touch inputs from mouse clicks and SDL controller input support," >}} and {{< gh-hovercard "9730" "two parameters:" >}} `-g`  for specifying a game file location and `-u` for specifying a user profile.
+
+Your writer also showed up, {{< gh-hovercard "9737" "updating" >}} the graphical filters and resolution multiplier options that were recently added to the Mainline yuzu.
+
+## Hardware section
+
+### NVIDIA driver 531.18, it’s a good one
+
+This latest (at the time of writing) release of the NVIDIA driver, 531.18, includes support for `VK_EXT_extended_dynamic_state3`, a Vulkan extension previous readers will be familiar with.
+Having this extension supported means fewer shaders need to be built, resulting in a smoother experience thanks to reduced shader stuttering, and lower RAM use.
+
+We recommend at least Turing and newer (GTX 1600, RTX 2000-4000) users to update, as we expect this release to not fix the issues affecting Pascal and Maxwell hardware (GTX 1000 and older).
+
+### AMD Ryzen 7000X3D series, and the importance of Game Mode
+
+AMD recently released their Zen 4 CPUs with 3D V-Cache technology, and in an effort to copy Intel in weirdness, they decided to release a top of the line asymmetric design with the 7950X3D.
+
+Only one of this gaming-beast CPU’s two CCDs (modules containing 8 cores each) has access to the expanded L3 V-Cache, and the decision on which one to use is up to Windows Game Bar. 
+To our dismay, there is no hardware scheduler here. 
+The chipset driver and a Windows toggle decides which mode is better.
+
+Since yuzu and basically any other emulator are not registered as games by Windows, the user will have to manually intervene and enable `Game Mode` in order to let the chipset driver profile performance and decide which CCD will provide the best performance.
```

Comment:
```suggestion
Since yuzu, and basically any other emulator, are not registered as games by Windows, the user will have to manually intervene and enable `Game Mode` in order to let the chipset driver profile performance and decide which CCD will perform the best.
```